app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt

```diff
@@ -0,0 +1,126 @@
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.anytypeio.anytype.R
+
+@Composable
+fun PulsatingCircleScreen() {
+    val context = LocalContext.current
+
+    var initialAnimationFinished by remember { mutableStateOf(false) }
+
+    var circleSize by remember { mutableStateOf(16.dp) }
+
+    LaunchedEffect(Unit) {
+        animate(
+            initialValue = 16f,
+            targetValue = 128f,
+            animationSpec = tween(durationMillis = 1000, easing = FastOutLinearInEasing)
+        ) { value, _ ->
+            circleSize = value.dp
+        }
+        initialAnimationFinished = true
+    }
+
+    val infiniteTransition = rememberInfiniteTransition(label = "")
+    val animatedSize by infiniteTransition.animateValue(
+        initialValue = 64.dp,
+        targetValue = 128.dp,
+        Dp.VectorConverter,
+        animationSpec = infiniteRepeatable(
+            animation = tween(1000, easing = FastOutLinearInEasing),
+            repeatMode = RepeatMode.Reverse
+        ), label = ""
+    )
+
+    val finalSize = if (initialAnimationFinished) animatedSize else circleSize
+    val finalBrush =
+        if (circleSize < 17.dp) SolidColor(Color(context.getColor(R.color.glyph_inactive))) else Brush.radialGradient(
+            colors = listOf(
+                Color.White.copy(alpha = 0.0f),
+                Color(context.getColor(R.color.glyph_inactive))
+            )
+        )
+
+    Column(
+        modifier = Modifier.fillMaxSize(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center
+    ) {
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .fillMaxHeight(),
+            contentAlignment = Alignment.Center
+        ) {
+            SimpleCircleShape(
+                size = finalSize,
+                gradient = finalBrush
+            )
+        }
+    }
+}
+
+@Composable
+fun SimpleCircleShape(
+    size: Dp,
+    gradient: Brush,
+    borderWidth: Dp = 0.dp,
+    borderColor: Color = Color.LightGray.copy(alpha = 0.0f)
+) {
+    Column(
+        modifier = Modifier
+            .wrapContentSize(Alignment.Center)
+    ) {
+        Box(
+            modifier = Modifier
+                .size(size)
+                .background(brush = gradient, shape = CircleShape)
+                .border(borderWidth, borderColor)
+        )
+    }
+}
+
+@Preview(showBackground = true)
```

Comment:
we have a @DefaultPreviews annotation now for Previews

Reply:
changed on @DefaultPreviews